### DISCUSSION TITLE
Feature Store built with dbt & fal

### DISCUSSION THREAD

**Discussion by burkaygur** [dbt-labs/devhub-writing#87] [Guides and playbooks]
Hi folks! 

We just released our [feature store implementation](https://github.com/fal-ai/dbt_feature_store) which is a dbt package under the hood as outlined [here](https://github.com/fal-ai/fal/issues/91). We would love to share our story with the community of how we built it.

Briefly, we took the feature store concept, broke it down to its building blocks, and built most of it as a dbt package (much like the metrics layer implementation). We will soon add some additional functionality to [fal](https://github.com/fal-ai/fal) to enable a Python API. This will let users interact with the feature store through Jupyter or Hex notebooks as well.

With fal feature store, we believe that you can have a very solid set up for tabular ML model workflows right within your dbt project.

**Comment by sungchun12** [comment]
@krevitt This blog post will be a great continuation of the work we've been doing with: https://github.com/dbt-labs/dbt-technical-blog-writing/issues/72

I really like this theme emerging around "baton passes" in the modern data stack. More and more, machine learning workflows are becoming ergonomic across analytics and machine learning engineers.

**Comment by sungchun12** [comment]
@burkaygur I just realized this is a discussion topic. Can you submit an Issue to keep track of this?

**Reply by burkaygur** [reply]
On it 👍🏽
